src/test/java/trello/hooks/Hooks.java

```diff
@@ -1,4 +1,22 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
 package trello.hooks;
 
+/**
+ * Key class.
```

Comment:
Please change the documentation with the right class name.

Reply:
Thank you, this wrong was fixed.